js/sliderView.js

```diff
@@ -394,7 +388,7 @@ define([
       }
     }
 
-  });
+  };
```

Comment:
Can you remove the `;` from the end of both classes? They're not necessary after a code block.
```js
if () {
}

class Name {
}

for () {
}
```